{
    "id": "CVE-2018-19585",
    "cisaKEV": false,
    "reported_exploited": false,
    "exploit_maturity": "poc",
    "counts": {
        "public_exploit_count": 4
    },
    "timeline": {
        "nvd_published": "2019-05-17"
    },
    "exploits": [
        {
            "url": "https://gitlab.com/exploit-database/exploitdb/-/blob/main/exploits/ruby/webapps/49334.py",
            "name": "GitLab 11.4.7 - RCE (Authenticated) (2)",
            "source": "exploitdb",
            "date_added": "2020-12-24",
            "exploit_maturity": "poc"
        },
        {
            "url": "https://gitlab.com/exploit-database/exploitdb/-/blob/main/exploits/ruby/webapps/49257.py",
            "name": "GitLab 11.4.7 - Remote Code Execution (Authenticated) (1)",
            "source": "exploitdb",
            "date_added": "2020-12-14",
            "exploit_maturity": "poc"
        },
        {
            "url": "https://github.com/xenophil90/edb-49263-fixed",
            "name": "Fixed version of the Python script to exploit CVE-2018-19571 and CVE-2018-19585 (GitLab 11.4.7 - Authenticated Remote Code Execution) that is available at https://www.exploit-db.com/exploits/49263 (Python 3.9).",
            "source": "github",
            "date_added": "2021-02-08",
            "exploit_maturity": "poc"
        },
        {
            "url": "https://github.com/Algafix/gitlab-RCE-11.4.7",
            "name": "GitLab 11.4.7 RCE exploit with different reverse shells. CVE-2018-19571 + CVE-2018-19585",
            "source": "github",
            "date_added": "2021-04-11",
            "exploit_maturity": "poc"
        }
    ]
}